src/wing/CCPACSWingSection.h

```diff
@@ -56,7 +56,7 @@ class CCPACSWingSection : public generated::CPACSWingSection
     // Gets the section scaling
     TIGL_EXPORT CTiglPoint GetScaling() const;
 
-    // Setter for translation
+    // Setter for translation.
```

Comment:
Why was this added 🙂

Reply:
seems like its really important ;) - Sorry, I'll restore that line

Reply:
No its, fine. I should have been clear in mz comment that this does not require any change 🙂